core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/arrow/ArrowNltkSentimentOperator.java

```diff
@@ -101,6 +95,17 @@ public void open() throws TexeraException {
         outputSchema = transformToOutputSchema(inputSchema);
 
         cursor = OPENED;
+        List<String> args = new ArrayList<>(Arrays.asList(PYTHON, PYTHONSCRIPT, PicklePath));
+        ProcessBuilder processBuilder = new ProcessBuilder(args).inheritIO();
+        try {
+            // Start Flight server (Python process)
+            processBuilder.start();
+            // Connect to server
+            flightClient = FlightClient.builder(rootAllocator, lc).build();
+            System.out.println("Flight Client:\t" + new String(flightClient.doAction(new Action("healthcheck")).next().getBody(), StandardCharsets.UTF_8));
```

Comment:
comment out all unnecessary `System.out.println` and `print` function calls in the two files. But don't delete them because they can be useful for future debugging purposes.